lib/scrabble.rb

```diff
@@ -1,7 +1,48 @@
 class Scrabble
 
+  def split_word(word)
+    word.chars
+  end
+
+  def capitalize_letters(letters)
+    letters.map { |char| char.upcase }
+  end
+
+  def get_letter_scores(cap_letters)
+    cap_letters.map { |char| point_values[char] }
+  end
+
   def score(word)
-    1
+    return 0 if word.nil?
+    letters = split_word(word)
+    cap_letters = capitalize_letters(letters)
+    letter_scores = get_letter_scores(cap_letters)
+    letter_scores.sum
+  end
+
+  def apply_multipliers(letter_scores, multipliers)
+    letter_scores.size.times.collect { |i| letter_scores[i] * multipliers[i] }
+  end
+
+  def check_10_point_bonus(multiplied, letters)
+    if letters.count < 7
+      multiplied.sum
+    elsif letters.count >= 7
+      (multiplied.sum + 10)
+    end
+  end
+
+  def score_with_multipliers(word, multipliers, final_multiplier = 1)
```

Comment:
I'd love for you to identify ways you could refactor this method; it's a bit long and difficult to read.